config/tikv.toml

```diff
@@ -148,7 +148,8 @@ capacity = "8G"
 # bit smaller.
 # region-max-size = "144MB"
 # region-split-size = "96MB"
-region-split-size = "1MB"
+# region-max-size = "2MB"
```

Comment:
should we remove commented code?

Reply:
will change it using `sed -i`

Reply:
sh "sed -i 's/\\# region-max-size = \\\"2MB\\\"/region-max-size = \\\"2MB\\\"/' config/tikv.toml"
                            sh "sed -i 's/\\# region-split-size = \\\"1MB\\\"/region-split-size = \\\"1MB\\\"/' config/tikv.toml"